website/common/script/libs/getItemInfo.js

```diff
@@ -37,6 +37,13 @@ function isItemSuggested (officialPinnedItems, itemInfo) {
   }) > -1;
 }
 
+function pinIsTogglable (itemInfo) {
+  if (itemInfo.path === 'armoire' || itemInfo.path === 'potion' || itemInfo.type === 'debuffPotion') {
```

Comment:
This is also referenced in the api.
Maybe this could be centralised.

API: https://github.com/HabitRPG/habitica/blob/573e472077d24dcf45964c6e363ff8819635e2c1/website/common/script/ops/pinnedGearUtils.js#L147-L149